File: VB/ChartUpdateExample/Record.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace ChartUpdateExample
	Public Class Record
		Public Property Value() As Integer
		Public Property Argument() As Object
		Public Property Series() As Object

		Public Sub New(ByVal value As Integer, ByVal argument As Object, ByVal series As Object)
			Me.Value = value
			Me.Argument = argument
			Me.Series = series
		End Sub
	End Class
End Namespace

File: CS/ChartUpdateExample/Record.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ChartUpdateExample
{
    public class Record
    {
        public int Value { get; set; }
        public object Argument { get; set; }
        public object Series { get; set; }

        public Record(int value, object argument, object series)
        {
            this.Value = value;
            this.Argument = argument;
            this.Series = series;
        }
    }
}
